src/partisan_peer_service_client.erl

```diff
@@ -61,8 +61,8 @@ init([Peer, From]) ->
     case connect(Peer) of
         {ok, Socket} ->
             {ok, #state{from=From, socket=Socket, peer=Peer}};
-        _Error ->
-            {stop, normal}
+        {error, Reason} ->
```

Comment:
I'm not convinced about this change - why was this necessary?  If a normal stop is no longer returned, then a client that has a closed port can trigger a client that is connected to it to crash it's entire peer service manager.